base/applications/rapps/lang/tr-TR.rc

```diff
@@ -17,17 +18,17 @@ BEGIN
         MENUITEM SEPARATOR
         MENUITEM "&Çıkış", ID_EXIT
     END
-    POPUP "Y&azılım"
+    POPUP "&Yazılım"
```

Comment:
You've now introduced an accelerator collision with the next menu, `"&Yardım`, see below.

Reply:
what does accelerator collison means ?

Reply:
The `&` thingie. See for example:
https://reactos.org/wiki/Translation_Introduction#Common_Characters
https://learn.microsoft.com/en-us/windows/win32/menurc/about-menus#menu-access-keys

Reply:
You have single key combination for multiple menu actions, in this case pressing Alt+Y will choose either `Yazılım` or `Yardım`, depending on what is earlier in the menu.